Sources/KlaviyoSwift/LoggerClient.swift

```diff
@@ -10,5 +10,5 @@ import os
 
 struct LoggerClient {
     var error: (String) -> ()
-    static let production = Self(error: { message in os_log("%{public}@", type: .error) })
+    static let production = Self(error: { message in os_log("%{public}s", type: .error, message) })
```

Comment:
Fixes [this](https://klaviyo.tpondemand.com/entity/149794-missing-log-contents-in-os_log-error).